io/pci/pci_hotplug.py

```diff
@@ -58,10 +58,12 @@ def setUp(self):
             if not linux_modules.module_is_loaded("pnv_php"):
                 linux_modules.load_module("pnv_php")
         self.dic = {}
-        self.device = self.params.get('pci_devices', default=' ').split(" ")
-        self.count = int(self.params.get('count', default='1'))
-        if not self.device:
+        self.device = self.params.get('pci_devices', default=None)
+        if self.device:
+            self.device = self.device.split(" ")
+        else:
```

Comment:
the existing logic is taken care of cancelling if pci_devices are not given, then may I know why we are changing this ?

Reply:
self.device = self.params.get('pci_devices', default=' ').split(" ")  >> was failing with error if no pci device was given.

so first check if pci present and than split is the right way